core/trino-web-ui/src/main/resources/webapp-preview/src/App.tsx

```diff
@@ -11,41 +11,92 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import trinoLogo from './assets/trino.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { HashRouter as Router, Routes, Route, Navigate } from 'react-router-dom';
+import {
+  Box,
+  Button,
+  Container,
+  CssBaseline,
+  // Using MUI Grid v2 which will become stable in the next MUI release when v1 is deprecated
+  // https://mui.com/material-ui/migration/migration-grid-v2/
+  Unstable_Grid2 as Grid,
```

Comment:
I'd change the commit to:

// TODO: update when MUI is updated to version 6

Reply:
fixed